backend/alert/alerts.py

```diff
@@ -70,9 +70,37 @@ def send_alert(self, close_notification=False):
 
 class Email(Alert):
     def __init__(self, reg):
-        super().__init__(
-            "alert/email_alert.html", reg, close_template="alert/email_close_alert.html"
+        t = loader.get_template("alert/email_alert.html")
+        self.text = t.render(
+            {
+                "course": reg.section.full_code,
+                "brand": "Penn Course Alert",
+                "auto_resubscribe": reg.auto_resubscribe,
+                "course_event": '<span style="color:#78D381;">opened up</span>!',
+                "email_body": (
```

Comment:
I think `email_body` and `course_event` should still be in a template, I was more thinking about creating new templates that inherit using the [`extends` keyword](https://docs.djangoproject.com/en/3.1/ref/templates/language/)